src/index.html

```diff
@@ -1,13 +1,51 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="card container__card" data-qa="card">
+        <a href="#!" class="card__link">
+          <img
+            src="images/imac.jpeg"
+            alt="Moyo-Product-imac/APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__img"
+          >
+        </a>
+        <a href="#!" class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
```

Comment:
I think you need to add an empty line between multiline sibling blocks https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md (point 3). Like this:
```suggestion
    <div class="container">
      <div class="card container__card" data-qa="card">
        <a href="#!" class="card__link">
          <img
            src="images/imac.jpeg"
            alt="Moyo-Product-imac/APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
            class="card__img"
          >
        </a>
        
        <a href="#!" class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </a>
```